templates/admin/src/components/layout/layout.module.scss

```diff
@@ -13,7 +13,7 @@
       'content';
     grid-template-columns: 1fr auto;
     grid-template-rows: auto 1fr;
-    height: 100%;
+    height: 100vh;
```

Comment:
Il me semble que ce ne soit pas une bonne pratique le 100Vh, si le scroll n'est pas effectué sur la balise body, ça risque de poser problème pour le responsive. Pour ce qui est des iPhones / Safari, la barre de recherche ne disparait pas.

<img width="536" alt="image" src="https://github.com/user-attachments/assets/b32afcbc-3a7c-4afb-876a-74ffabee1b78">


Sources:
- [stackoverflow](https://stackoverflow.com/a/76909359)
- [IOS full height](https://www.sabhya.dev/handling-ios-safari-toolbar-for-full-height-web-content)

Reply:
Yes bien vu, après le BO n'est pas censé être responsive.

je change quand même pour éviter que des erreurs soient générées dues à ca

Reply:
En effet, j'ai oublié de le préciser, désolé !

Ce n'est potentiellement pas applicable dans le cas du BO.